crates/bevy_math/src/rotation2d.rs

```diff
@@ -81,8 +81,8 @@ impl Rotation2d {
 
     /// A counterclockwise rotation of π/4 radians.
     pub const FRAC_PI_4: Self = Self {
-        cos: std::f32::consts::FRAC_1_SQRT_2,
-        sin: std::f32::consts::FRAC_1_SQRT_2,
+        cos: core::f32::consts::FRAC_1_SQRT_2,
```

Comment:
We should prefer `core` over `std` whenever possible, especially in standalone crates like `bevy_math`.

Reply:
This is a good point! I myself have probably been guilty of importing from `std` where `core` would do 😅